platform/darwin/src/MGLStyle.h

```diff
@@ -442,6 +442,22 @@ MGL_EXPORT
  */
 - (void)removeImageForName:(NSString *)name;
 
+#pragma mark Managing a Style’s Transition Options
+
+/**
+ The duration in seconds to animate any changes to the style URL or to layout and paint attributes.
```

Comment:
Actually, #5665 makes it sound like we don’t have “diff and patch” functionality in mbgl yet. Has the situation changed since July?